metadata-db-app/src/migrations/051_add_created_at_to_granule_tables.clj

```diff
@@ -0,0 +1,24 @@
+(ns migrations.051-add-created-at-to-granule-tables
+  "Adds created_at column to granule tables."
+  (:require
+   [clojure.java.jdbc :as j]
+   [config.mdb-migrate-helper :as h]
+   [config.migrate-config :as config]))
+
+(defn- add-created-at
+  []
+  (doseq [t (h/get-granule-tablenames)]
+    (h/sql (format "alter table %s add created_at TIMESTAMP WITH TIME ZONE" t))))
+
+(defn up
+  "Migrates the database up to version 51."
+  []
+  (println "migrations.051-add-created-at-to-granule-tables up...")
+  (add-created-at))
+
+(defn down
+  "Migrates the database down from version 51."
+  []
+  (println "migrations.051-add-created-at-to-granule-tables down...")
+  (doseq [t (h/get-granule-tablenames)]
```

Comment:
Could you make a function of this just to be consistent with the style of (defn up?, either that or drop the add-created-at.